demos/popover/index.html

```diff
@@ -49,6 +50,17 @@
             trigger: '#triggerBottom'
         }).render();
 
+        var popoverBottom = new Y.Popover({
```

Comment:
Those three variables are used to helps the debug. But I am checking the other variable on the first Popover instance.